### PR TITLE
test(ll): make cypress tests compatible with deployed environments

### PR DESCRIPTION
**Pull request by y3rsh**
# Overview

Labware Library Cypress tests were able to run locally and in CI but some patterns did not allow for the same test to execute against a deployed environment.  Additionally there were no instructions or make targets to quickly run the tests against a deployed environment. 

## Test Plan and Hands on Testing

- [x] all the specs work in the Cypress GUI and through make for me locally 
- [ ] reviewer checkout this branch and `make -C labware-library test-e2e-remote REMOTE_BASE_URL='https://labware.opentrons.com'`
- [x] CI works. The success of the `labware library e2e tests` job in `Labware Library test, build, and deploy` proves the tests also still works against localhost 

## Review requests

- normal review of code 
- run locally for yourself to validate and tick the box 😊 

## Risk assessment

- None, test only

### CODE REVIEW

**Comment by y3rsh** [comment]
~~There might be an issue where the tests are not working in CI???
Will have to figure that out. `make test-e2e` is working locally for me.~~

Fixed. Removed the mock and replaced with real file read.

File: .github/workflows/ll-test-build-deploy.yaml [hunk 1]
```diff
@@ -115,10 +115,11 @@ jobs:
           yarn config set cache-folder ./.yarn-cache
           make setup-js
       - name: 'test-e2e'
+        working-directory: labware-library
         env:
           OT_LL_MIXPANEL_ID: ${{ secrets.OT_LL_MIXPANEL_ID }}
           OT_LL_MIXPANEL_DEV_ID: ${{ secrets.OT_LL_MIXPANEL_DEV_ID }}
-        run: make -C labware-library test-e2e
+        run: make labware-library test-e2e
```
Comment (y3rsh) [inline .github/workflows/ll-test-build-deploy.yaml]:
```suggestion
        run: make test-e2e
```

File: labware-library/vite.config.mts [hunk 2]
```diff
@@ -7,14 +7,6 @@ import postColorModFunction from 'postcss-color-mod-function'
...
 import lostCss from 'lost'
 
-const testAliases: {} | { 'file-saver': string } =
-  process.env.CYPRESS === '1'
-    ? {
-        'file-saver':
-          path.resolve(__dirname, 'cypress/mocks/file-saver.js') ?? '',
-      }
-    : {}
-
```
Comment (shlokamin) [inline labware-library/vite.config.mts:18]: these deletions make me so happy

**Review by shlokamin — APPROVED** [review]
love this. your file saver approach is honestly way better than the mocked file saver we used before, we should do the same thing for PD

**Pull request merged**